### ISSUE TITLE
In a vagrant VM ssh access from outside isn't working

### ISSUE THREAD

**Issue by rubo77** [ffnord/ffnord-puppet-gateway#84]
Using this vagrant installation for an example gateway SSH isn't working from outside the VM: https://github.com/ffnord/ffnord-example/issues/14

**Comment by rubo77** [comment]
I enabled iptables logging to syslog:

```
iptables -N LOGGING
iptables -A INPUT -j LOGGING
iptables -A LOGGING -m limit --limit 2/min -j LOG --log-prefix "IPTables-Dropped: " --log-level 4
iptables -A LOGGING -j DROP
```
source:  http://www.thegeekstuff.com/2012/08/iptables-log-packets/

 Here is a dump of my syslog: https://gist.github.com/rubo77/f997cb732d50c0c430b7

**Comment by rubo77** [comment]
Hier stehen ja alle firewall regeln: https://github.com/ffnord/ffnord-puppet-gateway/tree/master/files/etc/iptables.d

Ich hab mal DROP durch DROP-log ersetzt:

    sed -i s/DROP/DROP-log/g /etc/iptables.d/900-INPUT-drop

und die vm neu gestartet, aber das hat auch nichts verbessert.

**Comment by rubo77** [comment]
Access from local IPs is denied, which is not usefull in an experimental environment, so this will fix it there: rubo77@4804fb0
